openstates/scrape/base.py

```diff
@@ -66,6 +71,22 @@ def clean_whitespace(obj):
     return obj
 
 
+def get_random_user_agent():
```

Comment:
I think this is a good approach for now, but in the future it might be annoying to have to update core if want to change this pool. An env var seems like it would be a monster mess. I kinda hate to go adding config files but maybe there's some better way to define in this in the future.

Reply:
I agree, this starting point is not optimal. Worth going as far as pulling a library in (https://pypi.org/project/fake-useragent/) ?